content/hi/transport-layer-security.md

```diff
@@ -0,0 +1,29 @@
+---
+title: ट्रांसपोर्ट लेयर सिक्योरिटी Transport Layer Security (TLS)
+status: Completed
+category: अवधारणा
+tags: ["सिक्योरिटी", "नेटवर्किंग"]
+---
+
+## यह क्या है
+
+ट्रांसपोर्ट लेयर सिक्योरिटी (टीएलएस) एक प्रोटोकॉल है जिसे नेटवर्क पर संचार को बढ़ी हुई सुरक्षा प्रदान करने के लिए डिज़ाइन किया गया है।
+यह इंटरनेट पर भेजे गए डेटा की सुरक्षित डिलीवरी सुनिश्चित करता है,
+जैसे डेटा की संभावित निगरानी और/या परिवर्तन से बचानाा।
+यह प्रोटोकॉल मैसेजिंग, ई-मेल आदि जैसे अनुप्रयोगों में व्यापक रूप से उपयोग किया जाता है।
+
+## यह किस समस्या का समाधान करता है
+
+टीएलएस के बिना, ब्राउज़िंग आदतें, ई-मेल पत्राचार, ऑनलाइन चैट और कॉन्फ्रेंसिंग कॉल जैसी संवेदनशील जानकारी
+ट्रांसमिशन के दौरान दूसरों द्वारा आसानी से पता लगाया और संशोधित किया जा सकता है।
+टीएलएस का समर्थन करने के लिए सर्वर और क्लाइंट एप्लिकेशन को सक्षम करना यह सुनिश्चित करता है
+उनके बीच प्रसारित डेटा को गोपित रखता है और डेटा तीसरे पक्ष द्वारा देखने योग्य नहीं है।
```

Comment:
Reading fluency of these phrases are not matching with each other.

Reply:
yes not matching any suggestions how to solve

Reply:
It try to match them

Reply:
I have update the lines